backend/src/main/java/com/imsnacks/Nyeoreumnagi/weather/service/Briefing.java

```diff
@@ -61,9 +61,13 @@ public static String buildNoRiskWelcomeMsg(int hour) {
         else return GOOD_EVENING_NO_RISK;
     }
 
-    public static String buildWeatherMsg(final WeatherRisk risk) {
-        final String from = getClockHourAsString(risk.getStartTime()); // <오전/오후> <1-12>시
-        final String to = getClockHourAsString(risk.getEndTime());
+    public static String buildWeatherMsg(LocalDateTime now, WeatherRisk risk) {
+        LocalDateTime start = risk.getStartTime();
+        LocalDateTime end = risk.getEndTime();
+        boolean isBeforeNow = start.isBefore(now);
+
+        final String from = getClockHourAsString(isBeforeNow ? now : start);
+        final String to = getClockHourAsString(end);
```

Comment:
P3: 이렇게 고치면 한번만 true/false 체크를 할 수 있지 않을까 생각했습니다 ㅎㅎ
근데 작성하신 코드가 사람이 이해하기에는 더 좋은 것 같네요!

```suggestion
        LocalDateTime start;
        LocalDateTime end = risk.getEndTime();
        
        if( start.isBefore(now) ) {
            start = risk.getStartTime();
        } else {
            start = now;
        }
        final String from = getClockHourAsString(start);
        final String to = getClockHourAsString(end);
```

Reply:
```java
LocalDateTime start = risk.getStartTime();
LocalDateTime end = risk.getEndTime();

final String from = getClockHourAsString(start.isBefore(now) ? now : start);
final String to = getClockHourAsString(end);
```
true/false 체크를 한번만 하려면 isBeforeNow을 굳이 할당하지 말고 삼항연산자 안에 넣어버려도 괜찮을거같아요 ㅎㅎ 백엔드 코드 컨벤션은 잘 모르지만...

Reply:
피드백 반영해서 평가식을 연산자 내부에 넣어 단순하게 바꾸었습니다. 두 분 피드백 감사합니다:)